index.js

```diff
@@ -1,17 +1,56 @@
 class SortedList {
-  constructor() {}
-
-  add(item) {}
-
-  get(pos) {}
-
-  max() {}
+  constructor(items,length) {
+    this.items = [];
+    this.length = [];
+  }
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    if(this.length > 1){
+      this.items.sort((a,b)=> a-b);
+      this.length ++;
+    }
+  }
+  get(pos) {
+    if (pos === null || pos > this.items.length){
+    throw new Error('OutOfBounds');
+  } else{
+    return this.items[pos];
+  }
+}
+  max() {
+    if (this.length == 0){
+    throw TypeError("EmptySortedList");
+    } else{
+    return Math.max.apply(null,this.items);
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.length == 0){
+      throw TypeError("EmptySortedList");
+      } else{
+      return Math.min.apply(null,this.items);
+      }
+  }
 
-  sum() {}
+  sum() {
+    if (this.items.length < 1) {
+      return 0;
+    } else {
+      return this.items.reduce((a, b) => {
+        return a + b;
+      });
+    }
+  }
 
-  avg() {}
+  avg() {
+    if (this.length == 0){
+      throw TypeError("EmptySortedList");
+      } else{
+      return this.sum()/this.items.length;
```

Comment:
aqui tienes la propiedad length actualizada asi que no era necesario this.items.length